src/config/index.ts

```diff
@@ -15,4 +15,5 @@ export default {
   explorerUrl: Config.EXPLORER_URL,
   sentryDsn: Config.SENTRY_DSN,
   emailNotificationsApi: Config.EMAIL_NOTIFICATIONS_API,
+  airdropApi: Config.AIRDROP_API,
```

Comment:
The fact it's undefined doesn't break anything? I'd add to every `.env` file a record:
```
AIRDROP_API = ""
``` 
to be sure it's not an undefined

Reply:
done